diff --git a/files/image_config/copp/copp_cfg.j2 b/files/image_config/copp/copp_cfg.j2
index dcae06fc0..2bea5a485 100755
--- a/files/image_config/copp/copp_cfg.j2
+++ b/files/image_config/copp/copp_cfg.j2
@@ -31,18 +31,8 @@
 	    "queue4_group3": {
 		    "trap_action":"trap",
 		    "trap_priority":"4",
-		    "queue": "4",
-		    "meter_type":"packets",
-		    "mode":"sr_tcm",
-{% if DEVICE_METADATA is defined and DEVICE_METADATA['localhost'] is defined and DEVICE_METADATA['localhost']['type'] is defined and 'Mgmt' in DEVICE_METADATA['localhost']['type'] %}
-		    "cir":"300",
-		    "cbs":"300",
-{% else %}
-		    "cir":"100",
-		    "cbs":"100",
-{% endif %}
-		    "red_action":"drop"
-	    },
+		    "queue": "4"
+    },
 	    "queue1_group1": {
 		    "trap_action":"trap",
 		    "trap_priority":"1",
