diff --git a/src/flashrom/Makefile b/src/flashrom/Makefile
index 70365d648..60c16ef75 100644
--- a/src/flashrom/Makefile
+++ b/src/flashrom/Makefile
@@ -12,7 +12,7 @@ $(addprefix $(DEST)/, $(MAIN_TARGET)): $(DEST)/% :
 	pushd ./flashrom-$(FLASHROM_VERSION_FULL)
 
 	# Check out tag: tags/0.9.7
-	git checkout -b flashrom-src v$(FLASHROM_VERSION_FULL)
+	git checkout -b flashrom-src tags/v$(FLASHROM_VERSION_FULL)
 
 	# Apply patch series
 	stg init
